jbpm-wb-human-tasks/jbpm-wb-human-tasks-api/src/main/java/org/jbpm/workbench/ht/service/TaskService.java

```diff
@@ -59,6 +59,8 @@ void updateTask(String serverTemplateId, String containerId, Long taskId, Intege
 
     void delegate(String serverTemplateId, String containerId, Long taskId, String entity);
 
+    List<Long> checkAndDelegateTasks(String serverTemplateId, String[] containerId, String[] taskIds, String entity);
```

Comment:
I guess it still makes sense to keep the method name as delegate only, and suggestion to return a map instead with Map<TaskKey, Boolean> representing if the operation succeeds, wdyt?

Reply:
I changed this a bit, the name DelegateTasks, that receives an array of taskKey to delegate. It returns the assignments array to be able to show the unsuccess notification in case the task coudn't be delegated and show the error in case any exception is shown. I think this way covers the whole situations..